File: Modul 6/Python/PRAK604-2310817310010-AkmallullailSyaban.py
str1 = input()
str2 = input()

if len(str1) != len(str2):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    pagar = 0
    asterik = 0
    for i in range(len(str1)):
        if str1[i] == " " and str2[i] == " ":
            print(" ", end="")
        elif str1[i] != str2[i]:
            print("#", end="")
            pagar += 1
        else:
            print("*", end="")
            asterik += 1
    print(f"\n* = {asterik}")
    print(f"# = {pagar}")
    if asterik >= pagar:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")

File: Modul 6/C/PRAK604-2310817310010-AkmallullailSyaban.c
#include <stdio.h>
#include <string.h>

int main() {
    char str1[99], str2[99];
    scanf("%[^\n]%*c", str1);
    scanf("%[^\n]%*c", str2);

    if (strlen(str1) != strlen(str2)) {
        printf("Panjang kalimat berbeda, pesan palsu");
    } else {
        int asterik = 0, pagar = 0;
        for (int i = 0; i < strlen(str1); i++) {
            if (str1[i] == ' ' && str2[i] == ' ') {
                printf(" ");
            } else if (str1[i] != str2[i]) {
                printf("#");
                pagar++;
            } else {
                printf("*");
                asterik++;
            }
        }
        printf("\n* =%d\n", asterik);
        printf("# =%d\n", pagar);
        if (asterik > pagar) {
            printf("Pesan Asli");
        } else {
            printf("Pesan palsu");
        }
    }
    return 0;
}